schema/checks.hcl

```diff
@@ -206,3 +206,44 @@ table "checks" {
     columns = [column.canary_id, column.type, column.name]
   }
 }
+
+table "check_statuses_aggregate" {
+  schema = schema.public
+  column "check_id" {
+    null = false
+    type = uuid
+  }
+  column "created_at" {
+    null = false
+    type = timestamp
+  }
+  column "interval_duration" { # Aggregate interval. Example: '1h', '24h'
```

Comment:
@moshloop I create just 1 table to store aggregated data for both 1h and 24h.

Reply:
@adityathebe Use this to add comments

https://atlasgo.io/atlas-schema/sql-resources#comment
```
    column "name" {
        type    = text
        comment = "A column comment"
    }
```

Reply:
@adityathebe we don't want 1 table for both it will perform very badly for queries and/or inserts - We also need an index on created_at (not a primary index) - Partitioning would fix the performance, but it adds unnecessary overhead now.